ui/src/app/dashboard/grouped-summary/grouped-summary.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {AggregationResponse} from '../../shared/model/AggregationResponse';
+import {JobStatus} from "../../shared/model/JobStatus";
+import {AggregationEntry} from "../../shared/model/AggregationEntry";
+
+@Component({
+  selector: 'jm-grouped-summary',
+  templateUrl: './grouped-summary.component.html',
+  styleUrls: ['./grouped-summary.component.css']
+})
+export class GroupedSummaryComponent implements OnInit {
+  @Input() aggregationResponse: AggregationResponse;
+
+  statusArray: Array<JobStatus> = [];
+
+  constructor() {
```

Comment:
I don't think you need to include empty constructors (I may be wrong on this, Angular compiler should complain if so)